logging/cluster-logging-release-notes.adoc

```diff
@@ -46,6 +48,8 @@ This release includes link:https://access.redhat.com/errata/RHSA-2022:6344[OpenS
 
 include::modules/cluster-logging-rn-5.5.adoc[leveloffset=+1]
 
+//include::modules/cluster-logging-rn-5.4.7.adoc[leveloffset=+1]
```

Comment:
You've commented out including the 5.4.7 RNs, but are still updating the file in this PR. Are the 5.4.7 changes going to be merged in later? I'm not sure what's going on.